libs/image/tests/test_image.cpp

```diff
@@ -388,7 +388,7 @@ TEST_F(ImageTest, Ktx) { // NOLINT
         const auto fileSize = getFileSize(path.c_str());
         ASSERT_GT(fileSize, 0);
         vector<uint8_t> buffer(fileSize);
-        std::ifstream in(path, std::ifstream::in);
+        std::ifstream in(path, std::ifstream::in | std::ifstream::binary);
```

Comment:
This fixes another Windows unit test issue

Reply:
Should it not by default assume it was meant to be binary ? or have I missed something?

Reply:
There is likely a \r\n sequence somewhere in the KTX file, which would get mistakenly coalesced on Windows.  This is a good fix.

Reply:
We had the same issue in other places.

Reply:
I put this PR on hold until after we move to GitHub Actions, but I can merge just this fix now.